---a/net/minecraft/block/BlockButtonWood.java
+++b/net/minecraft/block/BlockButtonWood.java
@@ -1,12 +1,11 @@
 package net.minecraft.block;
 
-public class BlockButtonWood extends BlockButton
-{
+public class BlockButtonWood extends BlockButton {
+
     private static final String __OBFID = "CL_00000336";
 
-    protected BlockButtonWood()
-    {
+    protected BlockButtonWood() {
         super(true);
     }
-}
+}
